Makefile

```diff
@@ -58,7 +58,7 @@ endif
 # Image URL to use all building/pushing image targets
 IMG ?= $(IMAGE_TAG_BASE):$(RELEASE_TAG)
 # ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
-ENVTEST_K8S_VERSION = 1.24
+ENVTEST_K8S_VERSION = 1.25.0
```

Comment:
I'm not sure we use this, but is this what the latest operator-sdk would scaffold?

Reply:
Yeah that's just the latest they can scaffold and is in the upgrade docs. We don't `use` it per say, but other cmds download that binary.